src/core/server/saved_objects/migrations/core/migration_es_client.ts

```diff
@@ -76,11 +76,11 @@ export function createMigrationEsClient(
   return methods.reduce((acc: MigrationEsClient, key: MethodName) => {
     set(acc, key, async (params?: unknown, options?: TransportRequestOptions) => {
       const fn = get(client, key);
-      if (!fn) {
+      if (!get(client, key)) {
```

Comment:
`fn` definition can be moved lower if you don't use here

Reply:
Good call.